src/command/mod.rs

```diff
@@ -19,11 +19,11 @@ pub enum AtuinCmd {
 }
 
 impl AtuinCmd {
-    pub async fn run(self) -> Result<()> {
+    pub fn run(self) -> Result<()> {
         match self {
-            Self::Client(client) => client.run().await,
+            Self::Client(client) => client.run(),
             #[cfg(feature = "server")]
-            Self::Server(server) => server.run().await,
+            Self::Server(server) => server.run(),
```

Comment:
This is a side change. The client does no `tokio::spawn` calls, so it doesn't need to be multi-threaded like the server. So this defers the tokio setup to the client/server respectively.